Resources/Textures/_Harmony/Objects/Magic/magicactions.rsi/meta.json

```
@@ -0,0 +1,14 @@
+{
+  "version": 1,
+  "license": "CC-BY-NC-SA-4.0",
+  "copyright": "Created by TheSecondLord (Github)/livelaughlord (Harmony discord) for the Harmony server.",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "item_curse"
+    }
+  ]
+}
```

Convention says that RSIs should use for space indentation
```suggestion
{
    "version": 1,
    "license": "CC-BY-NC-SA-4.0",
    "copyright": "Created by TheSecondLord (Github)/livelaughlord (Harmony discord) for the Harmony server.",
    "size": {
        "x": 32,
        "y": 32
    },
    "states": [
        {
            "name": "item_curse"
        }
    ]
}
```